docs/README.md

```diff
@@ -152,25 +152,25 @@
     * [Health Check Library](extensions/healthcheck-library.md)
   * [CA Rotation in Extensions](extensions/ca-rotation.md)
   * Blob storage providers
-    * [`BackupBucket` resource](extensions/backupbucket.md)
-    * [`BackupEntry` resource](extensions/backupentry.md)
+    * [`BackupBucket` resource](extensions/extension-resources/backupbucket.md)
```

Comment:
Why this stuttering/duplication in the folder name? Remove the `extension-` prefix?

Reply:
Honestly I was split between `resources` and `extension-resources`. I agree removing it would be better.